Code-4-Community/speak-for-the-trees-frontend
# Update popup for open planting sites

## Why

[ClickUp Ticket](https://app.clickup.com/t/1dk42dc)

Update the popup for open planting sites.


## This PR

Show the nearby address for open planting sites, if the site has one (as shown when there is a tree).

Add a link to tree planting instructions for open planting sites by providing a message ("Want to plant a tree here? Submit a request to the city!") and link (https://www.cityofboston.gov/311/) to users who want to plant a new tree.


## Screenshots

Open planting site popup (no nearby address)
![no-address](https://user-images.githubusercontent.com/33704204/134061851-a78b576d-9145-40ff-94c5-153b65ece637.png)

Open planting site popup (with nearby address)
![address](https://user-images.githubusercontent.com/33704204/134061835-3fce3fcf-ed3c-4032-a3cd-7a77cdbbef5d.png)
 

## Verification Steps

Visual verification with multiple open planting site popups (both with and without nearby addresses) on Firefox and Chrome. Ran `npm run check` and `npm run test`.
